### PR TITLE
Capitalized `Tokens`

### PR DESCRIPTION
**Pull request by RichardLitt**
License: MIT
Signed-off-by: Richard Littauer <richard.littauer@gmail.com>

### CODE REVIEW

File: core/commands/refs.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ Note: list all refs recursively with -r.
 		cmds.StringArg("ipfs-path", true, true, "Path to the object(s) to list refs from.").EnableStdin(),
 	},
 	Options: []cmds.Option{
-		cmds.StringOption("format", "Emit edges with given format. tokens: <src> <dst> <linkname>."),
+		cmds.StringOption("format", "Emit edges with given format. Tokens: <src> <dst> <linkname>."),
```
Comment (dignifiedquire) [inline core/commands/refs.go]: Shouldn't it be sth like `Available Tokens` or `Possible Tokens`?

Comment (RichardLitt) [inline core/commands/refs.go]: Yes. Happy to add that, too.

**Comment by whyrusleeping** [comment]
@RichardLitt RFM?

**Comment by RichardLitt** [comment]
@whyrusleeping It is now.

**Pull request merged**